### PR TITLE
Initial Fork From Litecoin

### PR DESCRIPTION
**Pull request by billy-mcfarland**
We're going to take every measure to make this right for everybody now, and make this right for everybody next year, on a large scale.

### CODE REVIEW

**Comment by ja-rule** [comment]
Very clean code, bro. We're gonna change the world. Do something that's literally never been done before.

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Translations
 pull from Transifex would automatically overwrite them again.
 
 We only accept translation fixes that are submitted through [Bitcoin Core's Transifex page](https://www.transifex.com/projects/p/bitcoin/).
-Translations are converted to Litecoin periodically.
+Translations are converted to Tridentlayers periodically.
```
Comment (ja-rule) [inline README.md:69]: nice...

**Comment by billy-mcfarland** [comment]
Changed the coinbase maturity to 10 so that people can spend their earned coins faster. We don't want some slow ass coin.

**Comment by ja-rule** [comment]
> Changed the coinbase maturity to 10 so that people can spend their earned coins faster. We don't want some slow ass coin.

I'm right there with you brother. Lightning fast, just like us.

**Comment by ja-rule** [comment]
@billy-mcfarland, Love the new changes man. Also, glad to see this launched too. With all the Tridentlayers we'll be making, we're gonna be fuckin like pornstars.

**Pull request merged**